link_linux.go

```diff
@@ -13,6 +13,14 @@ import (
 var native = nl.NativeEndian()
 var lookupByDump = false
 
+var macvlanModes = [...]uint32{
```

Comment:
Hehe, actually I already broke that convention in ipvlan patch :) I personally prefer to keep code simpler. But it is up to @vishvananda of course